src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx

```diff
@@ -189,8 +185,9 @@ const CheckInConfirmation = props => {
         ) : (
           <TravelPayReimbursementLink />
         )}
-        {/* Commenting out temporarily see: https://github.com/department-of-veterans-affairs/va.gov-team/issues/48126 */}
-        {/* <BackToAppointments appointments={appointments} /> */}
+        {appointments.length > 1 && (
+          <BackToAppointments appointments={appointments} />
```

Comment:
we are passing an appointments prop here but not doing anything with it or acknowledging it at all in the BackToAppointments component. We should be able to remove the prop here.